### DISCUSSION TITLE
From EBSD to Mesh to SIM

### DISCUSSION THREAD

**Discussion by bibiboom89** [neperfepx/neper#813] [Q&A] [answered]
Hi,

I'm trying to perform an FEM simulation from an EBSD map.
Using MATLAB and MTEX, I was able to generate a Raster Tessellation file (cf. attachment).
The tesr file only contains the dimension grid and the cell orientations.
All cells share the same phase.
[Files.zip](https://github.com/neperfepx/neper/files/14171696/Files.zip)

Here is the image produced with the following command:
neper -V EBSD_example.tesr -datavoxcol ori -print img_example
![img_example](https://github.com/neperfepx/neper/assets/63070505/605b071c-18ea-4633-8505-ac801b399837)

From that point, I was wondering how I could generate a mesh to perform a simple simulation using FEPX as presented in https://fepx.info/doc/tutorials/simple_simulation.html#simple-simulation. Does FEPX supports structured mesh with hexagonal element ? And how could I get a structured mesh from my tesr file?

I thank you a lot in advance,
Regards,
Bibi

**Comment by rquey** [comment, marked as answer]
FEPX only uses 10-node tets. There would be different options to mesh this structure, but the first thing I notice is the "noise" along the vertical direction.  The structure does not seem to be very well defined...
